src/components/LineChart/LineChart.props.ts

```diff
@@ -0,0 +1,11 @@
+export interface IChartProps {
+    title: string;
+    width: number;
+    height: number;
+    data: IChart[];
+}
+
+export interface IChart {
+    time: Date;
```

Comment:
what if i wanted to show something other than date on the x axis? could we make this a bit more generic?